Phaethon/ExternalApi/Controllers/InvoiceController.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Core.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Web.Http;
+using Newtonsoft.Json;
+
+namespace ExternalApi.Controllers
+{
+    [RoutePrefix("Invoice")]
+    public class InvoiceController : ApiController
+    {
+        [Route("Create")]
+        [HttpPost]
+        public HttpResponseMessage Create([FromBody] Invoice invoice)
+        {
+            HttpClient client = new HttpClient();
+            var result = client.PostAsJsonAsync("http://localhost:64007/Invoice/Create", invoice).Result;
```

Comment:
There are some conerns about performance, regarding .Result.
Depending on situation, it may even lead to deadlock. but I guess this can be addressed in #16 
```suggestion
            public async HttpResponseMessage Create([FromBody] Invoice invoice)
        {
            HttpClient client = new HttpClient();
            var result = await client.PostAsJsonAsync("http://localhost:64007/Invoice/Create", invoice);
```